doc/nep1.rst

```diff
@@ -247,6 +247,10 @@ Coding Conventions
   variables remain immutable, and gives those who read the code a better idea
   of the code's purpose.
 
+- Do not use `float` for money.
```

Comment:
add rationale even if obvious for most; but also, I'm not sure if this should be mentioned at all, seems out of scope